Enhanced Log/enhancedsyslogsettings.txt

```diff
@@ -87,76 +87,77 @@ Other Highlight Color=Andere Hervorhebungsfarbe
 :end
 
 :enhancd_log_default_plug:
-> 'Default' resets the Configuration and Colors.  Click 'Apply' to save your changes.  Click 'Done' when you are finished with your changes.
+> 'Standard' setzt die Konfiguration und die Farben zurück. Klick auf 'Übernehmen' um die Änderungen zu speichern.  Klick auf 'Abschließen' wenn du mit deinen Änderungen fertig bist.
 :end
 
 :enhanced_log_color_error_plug:
-> Color to use for 'Error' highlighting.  You can use the color name or number.  The color number begins with a '#'.
+> Zu verwendende Farbe für die Hervorhebung von 'Fehler'.  Es kann der Name der Farbe oder der Farbcode verwendet werden.  Der Farbcode beginnt mit einem '#'.
 :end
 
 :enhanced_log_color_minor_plug:
-> Color to use for 'Minor Issues' highlighting.  You can use the color name or number.  The color number begins with a '#'.
+> Zu verwendende Farbe für die Hervorhebung von 'kleine Unzulänglichkeiten'.  Es kann der Name der Farbe oder der Farbcode verwendet werden.  Der Farbcode beginnt mit einem '#'.
```

Comment:
@vakilando wäre hier statt "kleine Unzulänglichkeiten" nicht etwas wie "nicht kritische Fehler" oder "kleinere Fehler" in die Richtung angebrachter?

Reply:
ja, "kleine Unzulänglichkeiten" klingt etwas unbeholfen.... Wie wäre es mit "unkritische Fehler"?
Soll ich meinen Request dann noch mal erneuern, oder wie geht das? Ich hab mit Github irgendwie immer noch Probleme mit dem hin-und-her-mergen....